teos/teosd.py

```diff
@@ -68,171 +56,291 @@ def get_config(command_line_conf, data_dir):
     return config
 
 
-def main(config):
-    global db_manager, chain_monitor
+class TeosDaemon:
+    """
+    The :class:`TeosDaemon` organizes the code to initialize all the components of teos, start the service, stop and
+    teardown.
 
-    try:
-        signal(SIGINT, handle_signals)
-        signal(SIGTERM, handle_signals)
-        signal(SIGQUIT, handle_signals)
+    Args:
+        config (:obj:`dict`): the configuration object.
+        sk (:obj:`PrivateKey`:): the ``PrivateKey`` of the tower.
+
+    Attributes:
```

Comment:
All attributes are missing a colon after the closing parenthesis

Reply:
Fixed in f3135e8.